view/src/components/Code.jsx

```diff
@@ -31,7 +36,9 @@ const DIFF_DIRECTORY = 'diffs';
  *                                              of the function ends.
  * @param {Number} [props.specification.calling] - Tuple (old, new) of line numbers where are called
  *                                                 next functions which should be highlighted
- *                                                 or undefined if it is differing function.
+ *                                                 or undefined if no functions are called.
+ * @param {boolean} props.specification.differing - True if the symbol is the differing one
+ *   and the syntax difference should be highlighted.
```

Comment:
Thanks, I have fixed it, so the text description starts at the same column.

To be honest, I have been inconsistent with the comment indentation in the viewer. I think I started by indenting the next line so the description starts at the same column. Later, I thought it could waste a lot of space (lines) if the description is long, so for newly added parameters I continued the description at the beginning of the line and sometimes I probably indented the next line by two spaces. I'm not really sure what is the best approach.

Reply:
Yeah, me neither. Let's keep it as it is for now.